src/models/service/transactions/requests.rs

```diff
@@ -12,15 +12,16 @@ pub struct ConfirmationRequest {
 pub struct MultisigTransactionRequest {
     pub to: String,
     pub value: String,
-    pub data: String,
+    pub data: Option<String>,
     pub nonce: String,
     pub operation: Operation,
     pub safe_tx_gas: String,
     pub base_gas: String,
     pub gas_price: String,
     pub gas_token: String,
-    pub refund_receiver: String,
-    pub contract_transaction_hash: String,
+    pub refund_receiver: Option<String>,
+    pub safe_tx_hash: String,
```

Comment:
Do the core service accept this?  else you need to annotate with serde that for serializing it, it should use a different key

Reply:
Good catch

Reply:
cc @Uxio0

Reply:
I verified it using the kotlin example we wrote some time ago: https://safe-team-rinkeby.staging.gnosisdev.com/app/#/safes/0xd6f5Bef6bb4acD235CF85c0ce196316d10785d67/transactions